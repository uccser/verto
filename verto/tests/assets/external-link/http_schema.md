Check out this [website](http://www.google.com).
