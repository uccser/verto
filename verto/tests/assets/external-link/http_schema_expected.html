<p>Check out this <a href="http://www.google.com" target="_blank">website</a>.</p>
